test/packages/good/data_stream/foo/manifest.yml

```diff
@@ -1,5 +1,6 @@
 title: Nginx access logs
 type: logs
+allow_routing: true
```

Comment:
As this is related to elasticsearch configuration, I think it could be at the root level of `elasticsearch`, even if it doesn't belong to the `elasticsearch.privileges` object https://github.com/elastic/package-spec/pull/327#discussion_r862843182.

We did something similar for the time series mode in https://github.com/elastic/package-spec/pull/357/files.

Reply:
@felixbarny is this change still required? Wdyt about placing this parameter under `elasticsearch` object?

Reply:
Yes, this is still required. Are all the blockers resolved now? I don’t care so much about where the flag is. Happy to change it to be under elasticsearch if you think that’s a better place.

Reply:
This blocker was solved https://github.com/elastic/kibana/issues/115032, not sure if there were any others.

Yes, please update the branch to put the new setting under `elasticsearch`. We removed the `versions` directory, so you will also need to move the changes to the new structure.

Reply:
I've moved it and renamed and split the property so that it's both more self-explanatory and flexible.